tests/PyzTest/Shared/Testify/_support/Helper/Environment.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * This file is part of the Spryker Suite.
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace PyzTest\Shared\Testify\Helper;
+
+use Codeception\Module;
+use Spryker\Shared\Kernel\CodeBucket\Config\CodeBucketConfig;
+use Spryker\Shared\Kernel\CodeBucket\Config\CodeBucketConfigInterface;
+
+class Environment extends Module
+{
+    /**
+     * @var string
+     */
+    protected const TESTING_APPLICATION_ENV_NAME = 'devtest';
+
+    /**
+     * @return void
+     */
+    public function _initialize(): void
+    {
+        $rootDir = realpath(__DIR__ . '/../../../../../../');
+        $applicationEnv = $this->getApplicationEnv();
+
+        defined('APPLICATION_ENV') || define('APPLICATION_ENV', $applicationEnv);
+        defined('APPLICATION_STORE') || define('APPLICATION_STORE', (isset($_SERVER['APPLICATION_STORE'])) ? $_SERVER['APPLICATION_STORE'] : 'DE');
```

Comment:
I think this option would be better: `defined('APPLICATION_STORE') || define('APPLICATION_ENV', 'GLOBAL');`

Reply:
removed this file